client/components/PubByline/PubByline.tsx

```diff
@@ -21,8 +21,13 @@ type Props = OwnPubBylineProps & typeof defaultProps;
 
 const PubByline = (props: Props) => {
 	const { pubData, hideAuthors = false, hideContributors = false } = props;
+	// If arcadia, use A-Z byline option
```

Comment:
Since we actually do want this logic in place everywhere we use a `Byline`, I suggest moving it into that component.